### ISSUE TITLE
Polygon (Gone) Update - Alpaca Pro Update

### ISSUE THREAD

**Issue by scottstables** [alpacahq/Momentum-Trading-Example#37]
can we have the code updated given that Polygon is now gone/deprecated ?

**Comment by js10222** [comment]
Been having the same issue myself

**Comment by sriramarunachalam** [comment]
@scottstables Yeah same here with me. It throws 'REST' object has no attribute 'polygon' error. Any idea what the workaround is?
